pkgs/by-name/gn/gnumeric/package.nix

```diff
@@ -27,17 +31,28 @@ stdenv.mkDerivation rec {
     sha256 = "r/ULG2I0DCT8z0U9X60+f7c/S8SzT340tsPS2a9qHk8=";
   };
 
+  postPatch = ''
```

Comment:
For GNOME packages, we prefer having all the phases in the bottom, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.